includes/discovery/sensors.inc.php

```diff
@@ -12,6 +12,17 @@
     d_echo($pre_cache);
 }
 
+if (isset($device['dynamic_discovery']['modules']['sensors'])) {
+    foreach ($device['dynamic_discovery']['modules']['sensors'] as $key => $data_array) {
+        foreach ($data_array as $data) {
+            foreach ((array)$data['oid'] as $oid) {
+                $tmp_name = $data['oid'];
+                $pre_cache[$tmp_name] = snmpwalk_cache_oid($device, $oid, $pre_cache[$tmp_name], $device['dynamic_discovery']['mib'], null, '-OeQUs');
```

Comment:
Done.

Walking a single entry is ok but I'm not sure how well the code will work against that yet. I've added support for passing a description as a string as well as saying which key in the array contains the description so it's possibly ok with single OIDs.